src/components/@common/card/card.tsx

```diff
@@ -11,13 +11,13 @@ export function Card(props: CardProps) {
       <BankName>{bank}</BankName>
       <ICcard />
       <NumberWrapper>
-        <NumberItem type="text">{cardNumber?.["first"]}</NumberItem>
-        <NumberItem type="text">{cardNumber?.["second"]}</NumberItem>
+        <NumberItem type="text">{cardNumber?.first}</NumberItem>
+        <NumberItem type="text">{cardNumber?.second}</NumberItem>
         <NumberItem type="password">
-          {"•".repeat(cardNumber?.["third"].length)}
+          {"•".repeat(cardNumber?.third.length)}
```

Comment:
```suggestion
          {"•".repeat(cardNumber.third.length)}
```

이렇게 되는게 맞겠쬬..? cardNumber가 만약 nullable 하다면 지금의 코드에서는 에러가 날꺼에요

Reply:
넴 맞습니다..!!! 수정했어요!👍👍